### PR TITLE
Fix progress indicator

### LINKED ISSUE #91: ProgressIndicator never removed
Hello,

I noticed an issue when using showImage while the previous image is still loading. It keeps the previous progressIndicator opened above the ImageView.

```
  @Override
  public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
    super.onViewCreated(view, savedInstanceState);
    mBigImage = view.findViewById(R.id.mBigImage);
    mBigImage.setProgressIndicator(new ProgressPieIndicator());
  }

  private void displayImage(Image image) {
    if (image.url != null) {
      mBigImage.setInitScaleType(BigImageView.INIT_SCALE_TYPE_CENTER_INSIDE);
      mBigImage.showImage(Uri.parse(image.url));
    }
  }
```
If I call displayImage too quickly and the first showImage is not finished, it will keep its progressindicator open indefinitely. I tried to call cancel on mBigImage before calling showImage but it didn't resolve the problem.

### PR DESCRIPTION
**Pull request by SmasSive**
This PR fixes #91 
What it does is to remove progress indicator if it's shown **before** showing the image.

### CODE REVIEW

File: BigImageViewer/src/main/java/com/github/piasy/biv/view/BigImageView.java [hunk 1]
```diff
@@ -312,6 +312,11 @@ public void showImage(Uri uri) {
     public void showImage(final Uri thumbnail, final Uri uri) {
         mThumbnail = thumbnail;
         mUri = uri;
+
+        if (mProgressIndicatorView != null && mProgressIndicatorView.getVisibility() == View.VISIBLE) {
+            removeView(mProgressIndicatorView);
```
Comment (Piasy) [inline BigImageViewer/src/main/java/com/github/piasy/biv/view/BigImageView.java]: After we remove it, we should set `mProgressIndicatorView` to `null`.

Besides, I think we could change all `mProgressIndicatorView.setVisibility(GONE)` calls into removeView and nullify.

Comment (SmasSive) [reply BigImageViewer/src/main/java/com/github/piasy/biv/view/BigImageView.java]: Done!

**Comment by SmasSive** [comment]
BTW, if you try manually the uploaded code you will see that some images are not loaded because a Glide error, I'm looking for that too but I think it belongs to another PR. If you don't think so, please tell me and I'll add it in this PR.

**Comment by Piasy** [comment]
I tried the uploaded code, but didn't reproduce that bug. You can open another PR.

Thanks for your contribution!

**Pull request merged**